commit 27c2a6afeaad78af7041f849faceab9c6fd7cfec
Author: James Hughes <james.hughes@raspberrypi.org>
Date:   Tue Apr 25 10:15:06 2017 +0100

    brcmfmac: Make skb header writable before use
    
    The driver was making changes to the skb_header without
    ensuring it was writable (i.e. uncloned).
    This patch also removes some boiler plate header size
    checking/adjustment code as that is also handled by the
    skb_cow_header function used to make header writable.
    
    Signed-off-by: James Hughes <james.hughes@raspberrypi.org>
    Acked-by: Arend van Spriel <arend.vanspriel@broadcom.com>
    Signed-off-by: Kalle Valo <kvalo@codeaurora.org>

Index: linuxtest/dummy/rpi_1489_27c2a6afeaad78af7041f849faceab9c6fd7cfec.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1489_27c2a6afeaad78af7041f849faceab9c6fd7cfec.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
